src/js/components/Pagination.jsx

```diff
@@ -19,6 +21,7 @@ const Pagination = (props) => {
         className={ lastPage ? 'usa-button-disabled' : '' }
         onClick={ e => { if(!lastPage) props.getNextPage(page) }}
       >Next</button>
+      {props.isFetching ? <LoadingIcon/> : null}
```

Comment:
Would this be better to show in the results, eg parsing errors or edits? Or does it cause more jumping around the page?